tools/all-properties-component.tsx

```diff
@@ -145,7 +146,7 @@ export default function() {
       width="calc(25% - 10px)"
       wordBreak="normal"
       wordWrap="break-word"
-      zIndex="1"
+      zIndex={1}
```

Comment:
If strings work for these types of props, I think we should support them in the type def, otherwise we are introducing a _seemingly_ breaking change (only for TS consumers).